paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/Form.kt

```diff
@@ -91,41 +102,28 @@ class FormViewModel(
             val element = when (type) {
                 Field.NameInput -> TextFieldElement(NameConfig()) // All configs should have the label passed in for consistency
                 Field.EmailInput -> TextFieldElement(EmailConfig())
+                Field.CountryInput -> DropdownElement(CountryConfig())
             }
             listCompleteFlows.add(element.isComplete)
-            listOfPairs.add(element.input.map { Pair(type.paymentMethodCreateParamsKey, it) })
+            listOfPairs.add(element.paymentMethodParams.map {
+                Pair(
+                    type.paymentMethodCreateParamsKey,
```

Comment:
I think that this property could just be `key` or `name`. I don't think the UI element necessarily needs to know about how the data it collects will be used.

Reply:
Refactored